website/content/docs/job-specification/template.mdx

```diff
@@ -84,15 +84,15 @@ refer to the [Learn Go Template Syntax][gt_learn] Learn guide.
 - `perms` `(string: "644")` - Specifies the rendered template's permissions.
   File permissions are given as octal of the Unix file permissions `rwxrwxrwx`.
 
-- `uid` `(int: -1)` - Specifies the rendered template owner's user ID. Negative
+- `uid` `(int: nil)` - Specifies the rendered template owner's user ID. Negative
   values will use the ID of the Nomad agent user.
```

Comment:
If we're changing this default to `nil` we should document what `nil` does too.

Reply:
Ah good point